source/fileUtils.py

```diff
@@ -0,0 +1,16 @@
+import os
```

Comment:
Thank you. Understood. While I don't necessarily want to go through the 
code base just adding/tweaking copyright headers, I'm making an effort 
to fix this for any files I touch.